node/config/types.go

```diff
@@ -61,6 +61,8 @@ type StorageMiner struct {
 	Fees          MinerFeeConfig
 	Addresses     MinerAddressConfig
 	DAGStore      DAGStoreConfig
+
+	SturdyDB
```

Comment:
This should be a named field, otherwise the fields will not be in their own toml section
```suggestion
	SturdyDB SturdyDB
```

Also for the main config I'd avoid codenames, those are usually going to confuse new users.

Could call this field `ClusterDB` / `ClusterCoordinator` / something that makes it clear that this is configuring a database that can be shared between many lotus-miner instances and is not an optional feature